index.bs

```diff
@@ -791,6 +714,18 @@ To <dfn>initialize WebTransport over HTTP</dfn>, given a {{WebTransport}} object
 :: A single duplex stream for sending and receiving datagrams over this session.
    The getter steps for the `datagrams` attribute SHALL be:
      1. Return [=this=]'s [=[[Datagrams]]=].
+: <dfn for="WebTransport" attribute>incomingBidirectionalStreams</dfn>
+:: Returns a {{ReadableStream}} of {{BidirectionalStream}}s that have been
+   received from the remote host.
+
+   The getter steps for the `incomingBidirectionalStreams` attribute SHALL be:
+
+     1. Return [=[[ReceivedBidirectionalStreams]]=].
+     1. For each bidirectional stream received, create a corresponding
```

Comment:
It's weird that this step is here. The getter should have no side-effects, so bidirectional streams should be added to [[ReceivedBidirectionalStreams]] regardless of whether the getter was accessed.

Anyway, this is a pre-existing problem so no need to fix it in the this PR.

Reply:
Yeah, I agree. Fixing it would be easier with having this in WebTransport interface.